### ISSUE TITLE
Add footer.

### ISSUE THREAD

**Issue by anubhavpulkit** [anubhavsingh16/Amethi#7]
Add responsive footer for all pages.

**Comment by Muskan284** [comment]
I am working on it.

**Comment by Idadelveloper** [comment]
ok

**Comment by emrizkiem** [comment]
I would like to work on this issue. Could you assign me for this issue?

**Comment by anubhavpulkit** [comment]
> I would like to work on this issue. Could you assign me for this issue?

Yes please, I can't assign anyone, if you want to work on this project you are welcome. there is one more issue to add navbar.
